src/main/java/com/snowflake/kafka/connector/records/SnowflakeMetadataConfig.java

```diff
@@ -25,6 +26,7 @@ public SnowflakeMetadataConfig(Map<String, String> config) {
     // these values are the default values of the configuration
     boolean createtime = true;
     boolean topic = true;
+    boolean version = true;
```

Comment:
keeping this true will be a behavior change. Downstream application who are parsing this metadata will find this alarming. 
Lets keep this false as default.

Reply:
We can do the behavior change, it's not a big deal for connectors

Reply:
setting it to false makes sense. no need for behavior change then. resolving.